node-tests/fixtures/util-test/rfc232.js

```diff
@@ -1,7 +1,7 @@
 import fooBar from 'my-app/utils/foo-bar';
 import { module, test } from 'qunit';
 
-module('Unit | Utility | foo bar', function() {
+module('Unit | Utility | foo bar', function(hooks) {
```

Comment:
`hooks` is unused I believe

Reply:
Right, which is why I removed it in my earlier PR.

This is a fixture that will be compared to blueprint output, and I was assuming it was intentional that the blueprint would have `hooks`. 

But yeah, possibly the blueprint should say: `function(/* hooks */) {` to make it clear that the argument is available if you want it but deliberately unused.